src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -57,6 +57,22 @@
       "The connection URI as supported by the official drivers. "
           + "eg: ``mongodb://user@pass@locahost/``.";
 
+  public static final String OUTPUT_FORMAT_KEY_CONFIG = "output.format.key";
```

Comment:
As converters can be configured on both the key and value. I decided to add support for configuring output formats for both the key or the value.